ci/azure-release.yml

```diff
@@ -3,20 +3,10 @@ jobs:
     displayName: ${{ parameters.displayName }}
     condition: and(succeeded(), contains(variables['Build.SourceBranch'], '${{parameters.branchref}}'))
     steps:
-      - task: Docker@1
-        displayName: ${{ parameters.displayName }}
+      - script: |
+          docker run -i -e OWNER="managerforlustre" -e PROD="${{parameters.release}}" -e PROJECT="${{parameters.project}}" -e PACKAGE="${{parameters.package}}" -e SPEC="${{parameters.spec}}" -e KEY="$(KEY)" -e IV="$(IV)" -v $(pwd):/build:rw imlteam/copr
```

Comment:
Is it possible to add:
```
-e CARGO_HOME=/build/.cargo
```
?

Reply:
It's possible, but it won't buy as anything until azure has caching.